SD2019StackOverflowApp/src/main/java/ro/utcn/sd/mca/SD2019StackOverflowApp/controller/console/ConsoleController.java

```diff
@@ -0,0 +1,35 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp.controller.console;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.AnswerManagementService;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.QuestionManagementService;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.SOUserManagementService;
+
+//@Component
+@RequiredArgsConstructor
+@Profile("!test")
+public class ConsoleController implements CommandLineRunner {
```

Comment:
You should remove this class (but it is nice that you deactivated you by removing the component annotation).